docs/UserGuide.md

```diff
@@ -188,6 +188,8 @@ Edits an existing student in the address book.
 * At least one of the optional fields must be provided. You may refer to
 [Constraints of Add command](#constraints) for acceptable values of each field.
 * Existing values will be updated to the input values.
+* Command will behave weirdly for values after 2147483647.
```

Comment:
Can maybe either say that command will behave weirdly for index values after the given value and group together with the other stuff about the index or just put this into the notes about command format.

Reply:
Okays got it ty